src/tls/openssl.c

```diff
@@ -133,14 +133,60 @@ static void tls_context_destroy(void *ctx_backend)
     flb_free(ctx);
 }
 
+#ifdef _MSC_VER
+static int windows_load_system_certificates(struct tls_context *ctx)
+{
+    int ret;
+
+    HANDLE h;
```

Comment:
Could you please group variable definitions at the beginning of the function and use meaningful names for them?